packages/virtual/README.md

```diff
@@ -9,7 +9,8 @@
 [![version](https://img.shields.io/npm/v/@solid-primitives/virtual?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/virtual)
 [![stage](https://img.shields.io/endpoint?style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fsolidjs-community%2Fsolid-primitives%2Fmain%2Fassets%2Fbadges%2Fstage-0.json)](https://github.com/solidjs-community/solid-primitives#contribution-process)
 
-A basic [virtualized list](https://www.patterns.dev/vanilla/virtual-lists/) component for improving performance when rendering very large lists
+A headless `createVirtualList` utility function for [virtualized lists](https://www.patterns.dev/vanilla/virtual-lists/) and a basic, unstyled `VirtualList` component (which uses the utility).
```

Comment:
You could make `createVirtualList` and `VirtualList` into section headers and add hash links here like in other packages.

Reply:
this is what I meant:
```suggestion
- [`createVirtualList`](#createvirtuallist) - A headless  utility function for [virtualized lists](https://www.patterns.dev/vanilla/virtual-lists/)
- [`VirtualList`](#virtuallist) - a basic, unstyled component based on `createVirtualList`
```